applications/tari_app_grpc/proto/base_node.proto

```diff
@@ -223,3 +225,8 @@ message FetchMatchingUtxosRequest {
 message FetchMatchingUtxosResponse {
     TransactionOutput output = 1;
 }
+
+// This is the request type of the get all peers rpc call
+message GetPeersResponse{
+    repeated Peer peers = 1;
```

Comment:
Maybe this should be streamed?

Reply:
How big can this be?

Reply:
That is a good question, one that can be answered by writing a little test that serialises a peer and outputs the byte size, then expect 10's of thousands of peers (monero's grey list is 5000 big, but we also have client nodes (e.g. mobile wallets)). I recommend streaming because:
1. we can iterate through the peers instead of loading all of them in memory at once
1. faster time to start receiving and processing peers
1. we dont have to worry about the maximum message size being reached (not sure what that is on tonic grpc)

I agree that this won't be an issue right now, but as the peer list grows this will slow down